### ISSUE TITLE
Is there a way to combine ring data and arch data?

### ISSUE THREAD

**Issue by imrefazekas** [vasturiano/three-globe#54]
Hello,

I would like to have rings appearing at the points where archs are touching the globe.
Similar to this: https://github.blog/2020-12-21-how-we-built-the-github-globe/

And haven't found any API to make it happen?

**Comment by vasturiano** [comment]
@imrefazekas have you seen this example on the globe.gl repo? 
https://globe.gl/example/emit-arcs-on-click/

Code is here: https://github.com/vasturiano/globe.gl/blob/5c471aea8e1272180ce426af46cc9cb576b4d799/example/emit-arcs-on-click/index.html#L34-L54

**Comment by imrefazekas** [comment]
Yes I saw it. What I need is to have multiple arcs at the same time. With different speed and number and I should be able to patch the circles to them.

**Comment by vasturiano** [comment]
@imrefazekas that should be easily doable, as a variation of the above example.
